devops/resources/backup-prod-mckinsey-mongo-to-s3.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -exuo pipefail
+
+cd $WORKSPACE/configuration
+pip install -r requirements.txt
+. util/jenkins/assume-role.sh
+
+assume-role ${ROLE_ARN}
+
+cd $WORKSPACE/sysadmin
+
+bash util/clone-mongo/clone-mongo.sh  \
+  --collections modulestore,modulestore.active_versions,modulestore.definitions,modulestore.location_map,modulestore.structures,fs.chunks,fs.files \
+  --from-db-hosts 10.0.92.210,10.0.90.108,10.0.91.205  \
```

Comment:
These IPs should be a Parameter not in the script.  It can be a simple string list, and you configure the list of IPS in the internal repo